src/components/progress-steps/utils.jsx

```diff
@@ -0,0 +1,55 @@
+import { cn } from '@/utilities/functions';
+
+/**
+ * Helper function to generate classes for different step variants.
+ *
+ * @param {'dot' | 'number' | 'icon'} variant     - The type of step indicator.
+ * @param {boolean}                   isCurrent   - Whether the step is the current step.
+ * @param {Object}                    sizeClasses - The size classes for different step sizes.
+ * @param {'sm' | 'md' | 'lg'}        size        - The size of the step indicator.
+ * @return {string} The combined class names.
+ */
+export const getVariantClasses = ( variant, isCurrent, sizeClasses, size ) => {
+	const baseClass = `absolute rounded-full transition-colors duration-500 ${ sizeClasses[ size ].dot }`;
```

Comment:
```suggestion
	const baseClass = cn( 'absolute rounded-full transition-colors duration-500', sizeClasses[ size ]?.dot );
```

Try not to use a template string, because if a user passes any non-existing prop (for this case size) it will return undefined if you use optional chaining for another case it will throw an error. If you use the `cn` function it will filter out falsy values.

Reply:
we already using cn() in below code